reporter/html/src/main/java/org/itsallcode/openfasttrace/report/html/view/html/OriginLinkFormatter.java

```diff
@@ -138,6 +138,6 @@ private static URI convertPathWithoutProtocolToUri(final String path)
 
     private static String unWindowsify(final String path)
     {
-        return path.replaceAll("\\\\", "/");
+        return path.replace("\\\\", "/");
```

Comment:
:+1: Good one. That is definitely more efficient for a literal. I think I instinctively thought that `replace` would only search for one occurrence like in grep without modifiers. Helps to read the API, I guess.